### ISSUE TITLE
Logo Proposition for Documentation Enhancement (#76)

### ISSUE THREAD

**Issue by tinaok** [xarray-contrib/xdggs#82]
Hi all,

I believe having a logo (similar to what [xvec](https://github.com/xarray-contrib/xvec) does) would really help illustrate the documentation #76  more effectively.

Does anyone have any suggestions or ideas for a potential design? It could add a nice visual touch to our project!

**Comment by annefou** [comment]
![image](https://github.com/user-attachments/assets/83a0e201-0263-428f-869b-7f47f74605a5)

**Comment by annefou** [comment]
![image](https://github.com/user-attachments/assets/df9f6532-dff2-446e-b38a-1b255f4a9dca)

**Comment by allixender** [comment]
How "high-resolution" should the logo be? I could make something more logo like?

![image](https://github.com/user-attachments/assets/bc382b76-d655-492c-b311-411a13e215ad)

**Comment by allixender** [comment]
![image](https://github.com/user-attachments/assets/e4415247-d028-4d49-903f-9507d67b33a7)

**Comment by keewis** [comment]
we also need a version that remains readable at small sizes, for example for the logo displayed at the repository overview. For inspiration on that, see [xarray's collection](https://github.com/xarray-contrib/xarray.dev/tree/main/public/Xarray-assets) (we don't have to be as thorough, though, as far as I remember that was created by a professional designer).

The logos so far look great, but I'm missing the relation to `xarray` (otherwise they could just as well be logos for DGGS implementing packages).

**Comment by allixender** [comment]
You are right. I think it would meaningful to somehow visualise the 1-D index array in conjunction with a global cell grid, either in an unfolding/decomposition way (like the current xarray xvec or zarr logos) or superimposed over a globe? I think a globe of some sort would be meaningful, though, after all DGGS are for spatial data on the globe (not just in some rectangular projected plane)

**Comment by allixender** [comment]
For example, one of the coarsest resolution, a gradient along the index id, and then a 1-D array with the same gradient, with max 10-20 cells so to say?

**Comment by keewis** [comment]
what about taking a level-0 grid, plot a (section of a) 3D globe of that with the pixel edges in a certain color, and at the side a 2D field with one axis in the same color?

**Comment by allixender** [comment]
> what about taking a level-0 grid, plot a (section of a) 3D globe of that with the pixel edges in a certain color, and at the side a 2D field with one axis in the same color?
> 
> Edit: (not sure if that's what you were suggesting)

Yes, but not a 2D-field, but the 1-D index array same color coded.

**Comment by keewis** [comment]
with 2D field I meant a variable like `temperature(cells, time)` (3D like in `xarray`'s logo might be confusing since space is usually seen as 2D or 3D).

Otherwise agreed, that might be good (but I won't insist on anything except that it should somehow icon-ify xarray + DGGS)

**Comment by strobpr** [comment]
Hi Alex, all,

don't get me wrong, I like the fancy graphics, but I would prefer a logo that is not too 'hexagon' centric. In that sense the previous one was more neutral (maybe too neutral wrt 'global').

Cheers

Peter


________________________________

**Comment by tinaok** [comment]
I’m trying to create a logo where the Earth's surface appears to peel away like apple skin, symbolizing it as a one-dimensional surface (neutral in the shape of a grid), rather than a traditional latitude-longitude space. I've tried using some AI image generators, but I can't get a better image than this apple
![スクリーンショット 2024-10-30 15 11 54](https://github.com/user-attachments/assets/2ff97496-875b-4da4-9b60-405d5f2c96d9)

**Comment by benbovy** [comment]
Here is a quick sketch, dggs-diverse and reusing the same colors and layout than the Xarray logo.

256px

<img src="https://private-user-images.githubusercontent.com/4160723/382419816-64c24bfe-718e-49eb-8fcc-b3ec7c6d50da.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzA1MDExNjEsIm5iZiI6MTczMDUwMDg2MSwicGF0aCI6Ii80MTYwNzIzLzM4MjQxOTgxNi02NGMyNGJmZS03MThlLTQ5ZWItOGZjYy1iM2VjN2M2ZDUwZGEucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTEwMSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDExMDFUMjI0MTAxWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9YWRmMzQ0Nzg1MjRmYmVmYjVlNTYzODNiMjU3NmMyYzMyZDVlOGI0YTRiMmVlNTE2ZjkwMDgxMmZlNzFlMmE1OSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.ZvNWGVBPgoaUAEUNhzXpI80HvM-f-Le_FcyqdbspgeM" width="256" />

128px

<img src="https://private-user-images.githubusercontent.com/4160723/382419816-64c24bfe-718e-49eb-8fcc-b3ec7c6d50da.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzA1MDExNjEsIm5iZiI6MTczMDUwMDg2MSwicGF0aCI6Ii80MTYwNzIzLzM4MjQxOTgxNi02NGMyNGJmZS03MThlLTQ5ZWItOGZjYy1iM2VjN2M2ZDUwZGEucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTEwMSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDExMDFUMjI0MTAxWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9YWRmMzQ0Nzg1MjRmYmVmYjVlNTYzODNiMjU3NmMyYzMyZDVlOGI0YTRiMmVlNTE2ZjkwMDgxMmZlNzFlMmE1OSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.ZvNWGVBPgoaUAEUNhzXpI80HvM-f-Le_FcyqdbspgeM" width="128" />

 64px

<img src="https://private-user-images.githubusercontent.com/4160723/382419816-64c24bfe-718e-49eb-8fcc-b3ec7c6d50da.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzA1MDExNjEsIm5iZiI6MTczMDUwMDg2MSwicGF0aCI6Ii80MTYwNzIzLzM4MjQxOTgxNi02NGMyNGJmZS03MThlLTQ5ZWItOGZjYy1iM2VjN2M2ZDUwZGEucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTEwMSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDExMDFUMjI0MTAxWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9YWRmMzQ0Nzg1MjRmYmVmYjVlNTYzODNiMjU3NmMyYzMyZDVlOGI0YTRiMmVlNTE2ZjkwMDgxMmZlNzFlMmE1OSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.ZvNWGVBPgoaUAEUNhzXpI80HvM-f-Le_FcyqdbspgeM" width="64" />

(Still a bit too detailed IMO... the 2nd order tessellation could be removed... this would also make the vector / svg export lighter, which is better for a logo).

**Comment by annefou** [comment]
@benbovy your images are "empty" for me. I like the idea to reuse the same colours and layout than the Xarray logo!

**Comment by benbovy** [comment]
@annefou Yes if noticed that too when seeing this thread from the github mobile app.

Here is the SVG version in case anyone wants to play around with it:

![xdggs_logo_test_edit](https://github.com/user-attachments/assets/87aff90e-c7fe-4b52-b053-80cb8cb69ccf)

(note: the cell boundaries have been rasterized to significantly reduce the number of elements drawn. I've used matplotlib / cartopy, which seem to draw all cells as independent features - thus duplicate edges - including the cells on the hidden sides of the spheres).

**Comment by tinaok** [comment]
We have incorporated @benbovy's suggestion and created a preliminary logo.
![xdggs_preliminary_logo](https://raw.githubusercontent.com/xarray-contrib/xdggs/main/docs/_static/logos/xdggs_logo.png)

Currently, the landing page still displays the xarray logo. Would it be possible to update it to the xdggs_preliminary_logo?

<img width="422" alt="スクリーンショット 2024-11-28 0 51 04" src="https://github.com/user-attachments/assets/35cb0d62-4bdf-4899-a04e-988213a0fe97">

**Comment by keewis** [comment]
> Would it be possible to update it to the xdggs_preliminary_logo?

I don't think that's possible, it seems that's the organization's profile image, not the repository's. For example, `xarray`'s repository displays the pydata icon, not the `xarray` icon.
